### PR TITLE
Bookstore: Use Redux in React components

### PR DESCRIPTION
**Pull request by zewdu444**
## Bookstore: Use Redux in React components

- [x] Redux integrated to component
- [x] Add book and remove Integrated
- [x] Filter by categories has been added 
- [x] Readme file has been updated

### CODE REVIEW

File: src/components/BookForm.js [hunk 1]
```diff
@@ -32,8 +39,32 @@ function BookForm() {
...
+                >
+                  {category.category}
+
+                </option>
+              ))
+        }
+          </select>
+        </label>
+        {' '}
         <input type="submit" value="ADD BOOK" />
```
Comment (FrederickMih) [inline src/components/BookForm.js]: - [ ] Based on the requirements of this project, this should be a button **<Button>**. The button should include an event handler to add a book to the `books` array(with attributes _title_ and _author_)

File: src/components/Book.js [hunk 2]
```diff
@@ -1,30 +1,114 @@
...
+                </p>
+                <p />
+                <button
+                  id={book.item_id}
+                  onClick={() => dispatch(removeBook(book.item_id))}
+                  type="button"
+                >
+                  Remove
+
+                </button>
```
Comment (FrederickMih) [inline src/components/Book.js:60]: :+1:

**Review by FrederickMih — CHANGES_REQUESTED** [review]
## Hi :raising_hand:  @zewdu444 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!





## Highlights :green_circle: 

- PR is professional :+1: 
- Descriptive README.MD file :+1: 
- Good commit history :+1: 
- Linter checks passed :heavy_check_mark: 


## Required Changes ♻️
**N/A**
_Check the comments under the review._

## Optional suggestions
**N/A**
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/BookForm.js (hunk 1 above)
Comment (zewdu444) [reply src/components/BookForm.js]: thanks for the feedback changed to button

**Review by OLIPLICHE — APPROVED** [review]
Hi @zewdu444, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Highlights
- Applied the change, the App is working fine👏 👏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**